custom/staking/src/lib.rs

```diff
@@ -1069,7 +1105,14 @@ decl_module! {
             let controller = ensure_signed(origin)?;
             let ledger = Self::ledger(&controller).ok_or(Error::<T>::NotController)?;
             let stash = &ledger.stash;
+
             ensure!(Self::is_controller_eligible(&controller), Error::<T>::NotCompliant);
```

Comment:
Should we be checking the stash for compliance or the controller?

Reply:
This line is removed in the other currently active PR. Validators register intent to validate but they won't participate in validation till the next era. Committee sets the compliance status in PermissionedValidators map. Validators are selected in the next era if they have passed KYC.